### ISSUE TITLE
Without Test Credentials App Crashes

### ISSUE THREAD

**Issue by amorfc** [nymtech/nym-vpn-client#715]
When user lands the app, there is no indication that points you have to go to settings and apply the credential’s, beside the mail tho.

When I tried to connect network either  Anonymous or Fast type app was not responding any touch action forever. After I terminated process I was able to try again. 

Since this is a test flow maybe not crucial but still we have to get our testers to test application properly, this experience will them away without testing maybe.

Thank you :)

**Comment by marcdbz** [comment]
Thanks for reporting this. In case of missing credentials, you should be prompted by the app upon hitting the `Connect` button. Did that not happen? Thanks

**Comment by amorfc** [comment]
> Thanks for reporting this. In case of missing credentials, you should be prompted by the app upon hitting the `Connect` button. Did that not happen? Thanks

Nope. App was not responding I had to terminate the process That is why I could not see the next steps.

**Comment by amorfc** [comment]
Mobile application and IOS

**Comment by marcdbz** [comment]
Thanks. Could you uninstall the app from your device and try again? I just tested with the TestFlight build #278.
- Upon hitting `Connect`, you'l be prompted for a credential. You can enter it manually or scan a QR code.
- Then hit `Connect` again, authorize to set up a VPN connection and you should be good to go

**Comment by marcdbz** [comment]
Hi amorfc, do you still experience the issue?

**Comment by amorfc** [comment]
I apologize that I could not find a time to check it I will let you know when I try.

**Comment by rachyandco** [comment]
@amorfc the iOS have changed a lot since https://nymvpn.com/en/download/ios
can you test with the latest version?

**Comment by marcdbz** [comment]
Hi @amorfc we'll close this issue given that the app changed significantly. Feel free to re-open it in case of issues. Thanks

**Comment by amorfc** [comment]
> Hi @amorfc we'll close this issue given that the app changed significantly. Feel free to re-open it in case of issues. Thanks

It is better to close this PR I agree I am not able to test the issue anymore. If I found new one I will let you know you guys.

Great job 👏
